### PR TITLE
Tab config event

### PR DESCRIPTION
**Pull request by InsaneZein**
Part of [RHCLOUD-27097](https://issues.redhat.com/browse/RHCLOUD-27097)

configure_events

### CODE REVIEW

**Comment by karelhala** [comment]
@InsaneZein looks like the tests needs a bit of adjustment

**Comment by InsaneZein** [comment]
I'm currently working on them

File: src/pages/Notifications/List/BundlePage.tsx [hunk 1]
```diff
@@ -45,29 +49,85 @@ export const NotificationListBundlePage: React.FunctionComponent<NotificationLis
...
+                title={ `${Messages.pages.notifications.list.title} | ${props.bundle.displayName}` }
+                subtitle={ <span>This service allows you to configure which notifications different
+                    users within your organization will be entitled to receiving. To do this, create behavior groups and apply
+                    them to different events. Users will be able to opt-in or out of receiving authorized event notifications in their
+                <a href={ localUrl(`/user-preferences/notifications/${props.bundle.name}`,
+                    getInsights().chrome.isBeta()) }> User Preferences</a>.</span> }
+                action={ eventLogButton() }
+            />
+
+            <TabComponent configuration={ props.children } settings={ props.children }>
```
Comment (aferd) [inline src/pages/Notifications/List/BundlePage.tsx:128]: where did this section go? I don't see the time config in the mocks but curious if that was on purpose or not.

Comment (InsaneZein) [reply src/pages/Notifications/List/BundlePage.tsx:128]: I think this is supposed to go to a different page. I don't remember where though. We might need a story for that.

**Comment by aferd** [comment]
looks good! just one picky thing would be to line up the bundle tabs with the tabs below, similar to this.. so its stacked. (I got this review when I worked those initial tabs, so just a guess they would want the same thing!)
Screen Shot 2023-09-21 at 4 12 12 PM

File: src/pages/Notifications/List/BundlePage.tsx (hunk 1 above)
Comment (aferd) [reply src/pages/Notifications/List/BundlePage.tsx:128]: yeah, worth reaching out to UXD.

**Comment by InsaneZein** [comment]
Good catch. I fixed it
image
****

File: src/pages/Notifications/List/BundlePage.tsx (hunk 1 above)
Comment (InsaneZein) [reply src/pages/Notifications/List/BundlePage.tsx:128]: I agree. Will do.

**Comment by aferd** [comment]
looks like you need to run a lint fix after that lgtm!

**Pull request merged**